.github/workflows/transifex-update-templates.yml

```diff
@@ -0,0 +1,38 @@
+name: Transifex
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+  build:
+    name: ${{ matrix.type }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        type: [update-templates]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Install dependencies
+        run: |
+          sudo apt-get -qq update
+          sudo apt-get install -y gettext translate-toolkit transifex-client
+      - name: Configure Transifex
+        run: |
+          echo $'[https://www.transifex.com]' > ~/.transifexrc
+          echo $'api_hostname = https://api.transifex.com' >> ~/.transifexrc
+          echo $'hostname = https://www.transifex.com' >> ~/.transifexrc
+          echo $'username = api' >> ~/.transifexrc
+          echo $'password = ${{ secrets.TRANSIFEX_API_KEY }}' >> ~/.transifexrc
+      - name: Setup git config
+        run: |
+          git config user.name "GitHub Actions Bot"
```

Comment:
Why you update user name and email it there isn't git change? 
Should be remove.

Reply:
This is a git setup. It will be required if git commit should be made from this PR. See no reason to add a condition to add git credentials when making commit only.

Reply:
Where you commit/push from this PR?